aiidalab_widgets_base/databases.py

```diff
@@ -152,8 +152,8 @@ class :class:`aiidalab_widgets_base.structures.StructureManagerWidget`.
 
     def __init__(
         self,
-        embedded: bool = True,
-        title: str = None,
```

Comment:
This type was not correct (missing Optional). We're currently not doing any actual typechecking so it's better to just remove it, wrong types are worse than no types.